continuationsPlugin/src/test/resources/StateMachineTwoContinuationsChainedTwoValPriorTwoValBetweenTwoValAfterChainIgnored.scala

```diff
@@ -72,13 +67,11 @@ package continuations {
           $continuation.$label match
             {
               case 0 =>
+                $continuation.I$0 = x
```

Comment:
I think though that you don't want to do this before you check the result, it seems to me that the behaviour is different now